tests/unit/modules/test_netconfig.py

```diff
@@ -0,0 +1,117 @@
+"""
+    :codeauthor: Gareth J. Greenaway <gareth@saltstack.com>
+"""
+# Import Python libs
```

Comment:
Let's remove these import header comments please, python, salt, salt testing.

Reply:
Should be removed now, unless you also wanted the file docstring removed